src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
Everisthing is Ok. But it would be better to [use named function expressions instead of function declaration ](https://mate-academy.github.io/style-guides/javascript.html#71-use-named-function-expressions-instead-of-function-declarations)